lighthouse-core/audits/dobetterweb/appcache-manifest.js

```diff
@@ -33,7 +33,8 @@ class AppCacheManifestAttr extends Audit {
       category: 'Offline',
       name: 'appcache-manifest',
       description: 'Avoids Application Cache',
-      helpText: 'Application Cache has been [deprecated](https://html.spec.whatwg.org/multipage/browsers.html#offline) by [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers). Consider implementing an offline solution using the [Cache Storage API](https://developer.mozilla.org/en-US/docs/Web/API/Cache).',
+      helpText: 'Application Cache is deprecated by Service Workers. ' +
```

Comment:
"is deprecated by Service Workers" sounds a little weird. "is deprecated and Service Workers better something something" or something?